### PR TITLE
request PID C0DE for SBL Brain dev board

### PR DESCRIPTION
**Pull request by RegHolmes**
SBL Brain is an open source, low cost, PIC32 microcontroller based dev board.

### CODE REVIEW

**Comment by Arachnid** [comment]
Thanks for your pull request!

Before we can approve it, there's one issue that needs resolving. Please read carefully the guide at http://pid.codes/howto/, specifically the section on creating an 'org' page. Once you've added an org page and correctly referenced it as the owner of this PID, we can go ahead and approve this pull request.

**Comment by RegHolmes** [comment]
Hi Arachnid,

So I had to submit 2 pull requests. I had brain farted on the org file on the first, then i squared my self away and submitted the second.

Everything should be there (PID & ORG).

Thanks a lot for doing this.

File: 1209/C0DE/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SBL Brain
+owner: KMRH Labs
```
Comment (Arachnid) [inline 1209/C0DE/index.md]: This needs to match the name of the directory your org file is in - in this case, 'KMRH-Labs'.

**Comment by RegHolmes** [comment]
I fixed the issue. Sorry for the confusion. Thanks again.

**Pull request merged**